backend/src/main/java/ca/bc/gov/app/repository/client/SubmissionRepository.java

```diff
@@ -1,9 +1,24 @@
 package ca.bc.gov.app.repository.client;
 
 import ca.bc.gov.app.entity.client.SubmissionEntity;
+import reactor.core.publisher.Mono;
+import java.time.LocalDateTime;
+import org.springframework.data.r2dbc.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.data.repository.reactive.ReactiveCrudRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface SubmissionRepository extends ReactiveCrudRepository<SubmissionEntity, Integer> {
+  
+  @Query(""" 

```

Comment:
This kind of query already exist by default without the use of `@Query` annotation. See [the spring boot documentation](https://docs.spring.io/spring-data/jpa/docs/current-SNAPSHOT/reference/html/#jpa.query-methods.query-creation) for details

Reply:
Ok. Done.